open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Is this is a title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>